### PR TITLE
Feature/pass props to router

### PR DESCRIPTION
**Pull request by mxaly**
What?
* accept optional props in `react_router` helpers
* pass props as payload to integration 
* detect if registered router is a router instance or a function
  * if function, pass props from payload to it and render result
  * if router instance, just render.

solving #3

### CODE REVIEW

**Comment by mxaly** [comment]
solving: #3

File: README.md [hunk 1]
```diff
@@ -34,6 +34,24 @@ RWRReactRouter.register('MyRouter', MyRouter);
 <%= react_router('MyRouter') %>
 ```
 
+### Passing props to router.
+`RWRReactRouter.register` accepts both - react-router instance and function.
+When using function, props passed through `react_router` helper or will be passed as argument:
```
Comment (mwsteb) [inline README.md]: What? This sentence could use some polish. I've no clue what it tries to convey.

**Comment by pjurczynski** [comment]
looks

**Comment by mwsteb** [comment]
No, it doesn't. It should read:
```
When registering function instead of react-router instance, integration will take props (if any were passed to `react_router` helper) and will pass them to given function as first argument. This way you can pass them down to router of you choice:
```

**Pull request merged**